scripts/helper/functions.sh

```diff
@@ -128,8 +128,17 @@ create_certificates()
 
   # Enable Docker appuser to read files when created by a different UID
   echo -e "Setting insecure permissions on some files in ${DIR}/../security for demo purposes\n"
+  if [[ "$OSTYPE" == "darwin"* ]]
+  then
+      # Mac OS
```

Comment:
Format for lines below -- needs indentation.

Reply:
done in 0c86b59f94d03ab709be905d35c8d1c5acba914a